src/server/migrations/1_create_initial_tables.sql

```diff
@@ -35,4 +35,16 @@ CREATE TABLE users (
 );
 
 ALTER TABLE edge
-ADD CONSTRAINT PK_edge PRIMARY KEY (source_id, target_id);
\ No newline at end of file
+ADD CONSTRAINT PK_edge PRIMARY KEY (source_id, target_id);
```

Comment:
The migration system change is now merged, so SQL changes have to be done as proper migrations.